guides/common/modules/proc_importing-templates-using-ansible.adoc

```diff
@@ -0,0 +1,18 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="importing-templates-using-ansible"]
+= Importing templates using Ansible
+
+You can import templates from a repository of your choice.
+You can use different protocols to point to your repository, for example `/tmp/dir`, `git://example.com`, `\https://example.com`, and `ssh://example.com`.
+
+.Procedure
+* Use the `{ansible-namespace}.templates_import` module.
+
+.Additional resources
+ifdef::satellite[]
+* {ansible-galaxy-doc}/templates_import/[{ansible-namespace}.templates_import module – Sync Templates from a repository]
```

Comment:
DITA does not support attributes as parts of links. You can use an attribute for an entire link.